### ISSUE TITLE
Occurrence result list, country: replace text for missing data

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#212]
**Occurrence result list, country: replace text for missing data**

&quot;country.ZZ&quot; shows up in place of a country name for cases where the country information is missing from records. Example: https://demo.gbif.org/occurrence/search?taxon_key=2802896; https://demo.gbif.org/occurrence/1141507401

Suggest to replace with a brief text string like &quot;(unspecified)&quot;

-----
fbitem-4cd51ab3c35a7b3305c9ec1422d26bc93a21a482
Reported by: @ahahn-gbif
System: Firefox 53.0.0 / Windows 10 0.0.0
Referer: https://demo.gbif.org/occurrence/search?taxon_key=2802896
Window size: width 1510 - height 690
[API log](http://elk.gbif.org:5601/app/kibana#/discover/UAT-Varnish-403s?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2017-06-13T09:52:03.366Z',mode:absolute,to:'2017-06-13T09:58:03.366Z'))&_a=(columns:!(request,response,clientip),filters:!(),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499%20AND%20(request:%22%2F%2Fapi.gbif.org%22)')),sort:!('@timestamp',desc))&indexPattern=uat-varnish-*&type=histogram)
[Site log](http://elk.gbif.org:5601/app/kibana#/discover/UAT-Varnish-403s?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2017-06-13T09:52:03.366Z',mode:absolute,to:'2017-06-13T09:58:03.366Z'))&_a=(columns:!(request,response,clientip),filters:!(),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E399%20AND%20(request:%22%2F%2Fdemo.gbif.org%22)')),sort:!('@timestamp',desc))&indexPattern=uat-varnish-*&type=histogram)

**Comment by MortenHofft** [comment]
The country code was missing in the translation file. I tried replacing it with "(unspecified)" but found that it was easier to spot those with missing if the field was empty instead. Will be in the next deployment. thanks

**Comment by MortenHofft** [comment]
I have removed it from the search and added ZZ as a key to the translation file. So now it shows as unknown. that isn't ideal either, but better at least. I have created an issue for the display on occurrences here:
https://github.com/gbif/portal-feedback/issues/221

as it looks now
![screen shot 2017-06-21 at 17 49 57](https://user-images.githubusercontent.com/6759267/27393914-c85b9ca0-56aa-11e7-9c0d-0c32b7c72784.png)

and 

![screen shot 2017-06-21 at 17 50 06](https://user-images.githubusercontent.com/6759267/27393917-cacfcb28-56aa-11e7-9295-ec56b60d2261.png)
